### DISCUSSION TITLE
听张学友最新的演唱会专辑显示无法播放

### DISCUSSION THREAD

**Discussion by verygoogle** [1715173329/UnblockNeteaseMusic#418] [Q&A]
食用方法是win10 docker 作法
发现的疑惑是用docker run -dt -p 5552:8080 --name music3 --restart=always -e ENABLE_FLAC=true -e ENABLE_LOCAL_VIP=true nondanee/unblockneteasemusic -p 8080:8081 -o kuwo qq netease kugou migu xiami baidu joox 可以正常播放张学友最新的演唱会专辑
而且观察替换源是kuwo 。用docker run -dt -p 5553:8080 --name music2 --restart=always -e ENABLE_FLAC=true -e ENABLE_LOCAL_VIP=true pan93412/unblock-netease-music-enhanced -p 8080:8081 -o kuwo qq kugou migu pyncmd bilibili joox就显示无法播放 是我哪些弄错了吗 请高手帮忙看看 切来切去有点麻烦

**Comment by pan93412** [comment]
你這兩個 port 似乎不一樣？此外可以把 `LOG_LEVEL` 環境變數開到 debug 模式，這樣可以了解實際情況。

**Comment by verygoogle** [comment]
port不一样并不影响 用5553播其他的正常 目前发现的就是张学友的最新专辑不行

**Reply by pan93412** [reply]
那我不太清楚。你先開看看 DEBUG 模式，然後幫我確定一下這個 UNM instance 有沒有收到資料，以及有沒有正確處理。

**Comment by verygoogle** [comment]
log里面无任何显示 软件界面显示无法添加播放

**Reply by pan93412** [reply]
那就異常了：代表這個 UNM 根本沒收到任何請求。

**Reply by pan93412** [reply]
你確定真的有加 `-e LOG_LEVEL=debug` 嗎？

**Reply by verygoogle** [reply]
又试了 加了debug参数  就显示 url: "localhost:8081"

**Comment by verygoogle** [comment]
我把环境参数去掉试下

**Reply by verygoogle** [reply]
去掉一样 不给播 好奇怪 试了林俊杰的付费专辑又能播

**Comment by verygoogle** [comment]
只有我有这个问题吗 /:捂脸

**Reply by pan93412** [reply]
啊！忘了說，我們的 TLS 憑證跟原版有別。你或許是忘記安裝並放行我們的 TLS 憑證。

**Reply by verygoogle** [reply]
我就照着教程运行docker 命令  该如何安装放行TLs  是不是安装ca证书呀 安装了也不行

**Comment by verygoogle** [comment]
今天又发现了一首歌 黄新淳的 run away 双击也是显示无法添加播放
